### PR TITLE
USHIFT-924: Add performance monitoring logs to VM

### PR DESCRIPTION
**Pull request by pacevedom**
Add refs to enable performance metrics collection in any job. By using performance co-pilot package now it is possible to enable performance metrics logging for later retrieval.
The pmlogs ref will retrieve all the raw data and place it under the artifacts subdir for later analysis with other tools.
Includes usage for microshift conformance reduced tests.

### CODE REVIEW

**Comment by pacevedom** [comment]
/pj-rehearse periodic-ci-openshift-microshift-main-periodics-e2e-reboot

**Comment by pacevedom** [comment]
/pj-rehearse periodic-ci-openshift-microshift-main-periodics-e2e-reboot

**Comment by pacevedom** [comment]
/pj-rehearse periodic-ci-openshift-microshift-main-periodics-e2e-reboot

**Comment by pacevedom** [comment]
/pj-rehearse periodic-ci-openshift-microshift-main-periodics-e2e-reboot

**Comment by pacevedom** [comment]
/pj-rehearse periodic-ci-openshift-microshift-main-periodics-e2e-reboot

**Comment by pacevedom** [comment]
/pj-rehearse periodic-ci-openshift-microshift-main-periodics-e2e-reboot

**Comment by pacevedom** [comment]
/pj-rehearse periodic-ci-openshift-microshift-main-periodics-e2e-reboot

**Comment by pacevedom** [comment]
/pj-rehearse pull-ci-openshift-microshift-main-e2e-openshift-conformance-reduced

**Comment by pacevedom** [comment]
/pj-rehearse pull-ci-openshift-microshift-main-e2e-openshift-conformance-reduced

File: ci-operator/step-registry/openshift/microshift/infra/pcp/openshift-microshift-infra-pcp-commands.sh [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+fi
+
+gcloud auth activate-service-account --quiet --key-file "${CLUSTER_PROFILE_DIR}"/gce.json
+gcloud --quiet config set project "${GOOGLE_PROJECT_ID}"
+gcloud --quiet config set compute/zone "${GOOGLE_COMPUTE_ZONE}"
+gcloud --quiet config set compute/region "${GOOGLE_COMPUTE_REGION}"
+
+LD_PRELOAD=/usr/lib64/libnss_wrapper.so gcloud compute --project "${GOOGLE_PROJECT_ID}" ssh \
+  --zone "${GOOGLE_COMPUTE_ZONE}" \
+  rhel8user@"${INSTANCE_PREFIX}" \
```
Comment (dhellmann) [inline ci-operator/step-registry/openshift/microshift/infra/pcp/openshift-microshift-infra-pcp-commands.sh:23]: Is `rhel8user` still correct for the RHEL 9 image?

**Comment by pmtk** [comment]
I think we need some dev docs on how to use PCP artifacts

File: ci-operator/step-registry/openshift/microshift/infra/pcp/openshift-microshift-infra-pcp-commands.sh (hunk 1 above)
Comment (pacevedom) [reply ci-operator/step-registry/openshift/microshift/infra/pcp/openshift-microshift-infra-pcp-commands.sh:23]: Missed this comment. Yes, it is still valid.

**Comment by pacevedom** [comment]
> I think we need some dev docs on how to use PCP artifacts

Let me create a task for that in USHIFT and have a special howto doc.

**Comment by pacevedom** [comment]
/pj-rehearse pull-ci-openshift-microshift-main-e2e-openshift-conformance-reduced

**Comment by pacevedom** [comment]
/pj-rehearse pull-ci-openshift-microshift-main-e2e-openshift-conformance-reduced

**Comment by pacevedom** [comment]
/pj-rehearse pull-ci-openshift-microshift-main-e2e-openshift-conformance-reduced

**Comment by pacevedom** [comment]
/pj-rehearse pull-ci-openshift-microshift-main-e2e-openshift-conformance-reduced-arm

**Comment by pacevedom** [comment]
/pj-rehearse pull-ci-openshift-microshift-main-e2e-openshift-conformance-reduced pull-ci-openshift-microshift-main-microshift-e2e

**Comment by ggiguash** [comment]
/pj-rehearse pull-ci-openshift-microshift-main-microshift-e2e

**Comment by pacevedom** [comment]
/pj-rehearse pull-ci-openshift-microshift-main-e2e-openshift-conformance-reduced

**Pull request merged**